mono/utils/bsearch.h

```diff
@@ -9,6 +9,10 @@
 
 #include "mono/utils/mono-compiler.h"
 
+#ifdef __cplusplus
```

Comment:
Use `G_BEGIN_DECLS` in all places you do that.

Reply:
I tried. It is not always available. I can try more.